test/functional/test_runner.py

```diff
@@ -86,7 +86,6 @@
     'feature_pruning.py',
     'feature_dbcrash.py',
     'feature_index_prune.py',
-    'wallet_pruning.py --legacy-wallet',
```

Comment:
56e9bef655673becb3e265948143a9e860a5c68b: nit: could also update the example in `test_runner.py`:
```py
for exclude_test in exclude_tests:
    # A space in the name indicates it has arguments such as "wallet_basic.py --descriptors"
    if ' ' in exclude_test:
```

Reply:
Done